recipe/meta.yaml

```diff
@@ -17,15 +17,15 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.6,<3.10
```

Comment:
remove here this pinning at all ... you can add instead a "skip: True. # [py<36]" in "build:"